src/Demos/HSP.lagda

```diff
@@ -1,104 +1,81 @@
-2021,\footnote{See the
+The \agdaalgebras library \cite{ualib_v2.0.1} formalizes the foundations of universal algebra
+in intensional Martin-Löf type theory (\mltt) using \agda~{Norell:2007,agdaref}.
+The library includes a wide collection of definitions and verified theorems that appropriately codify
+classical set-theory-based universal algebra and equational
+logic.
+
+The first major milestone of the \agdaalgebras project is a proof of \emph{Birkhoff's
+variety theorem} (also known as the \emph{HSP theorem})~\cite{Birkhoff:1935}.
+To the best of our knowledge, this constitutes the first machine-verified proof of Birkoff's
+celebrated 1935 result.
+
+The proof presented here is the second proof. The first proof%
+\footnote{See the
  \href{https://github.com/ualib/ualib.github.io/blob/71f173858701398d56224dd79d152c380c0c2b5e/src/lagda/UALib/Birkhoff.lagda}{Birkhoff}
  module of the
  \href{https://github.com/ualib/ualib.github.io/commit/71f173858701398d56224dd79d152c380c0c2b5e}{15
  Jan 2021  commit (71f1738)} of the
  \href{https://github.com/ualib/ualib.github.io}{ualib/ualib.gitlab.io}
  repository~\cite{ualib_v1.0.0}.}
 suffered from a few flaws that raised two concerns. First, it was not clear whether the
-proof was fully constructive. Second, it was shown that if one (say, a subversive
-adversary) were to take the type \ab{X}---which we use to represent an arbitrary collection of
-variable symbols---to be just the two element type, then one could combine this with our
-proof and derive a contradiction. To resolve these issues, as well as improve the
-\agdaalgebras library more generally, we have rewritten parts of the library and have
-developed a new formal proof of the HSP theorem. We are confident that the new
-proof---based on the \agdaalgebras library, ver.~2.0.1~\cite{ualib_v2.0.1}---is fully
-constructive and correct, a conviction we justify in due course (q.v.).
-
-What follows is a self-contained formal proof of the HSP theorem which is achieved by
-extracting into a single \agda module a subset of the \agdaalgebras library, including only
-the pieces needed for the proof.\footnote{The main body of the paper was generated by a literate
-\agda file (\HSPlagda) which others can type-check with \agda ver.2.6.2 and \agdastdlib ver.1.7 to
-verify its correctness.}
+proof was fully constructive (because of its use of univalence in \mltt). Second,
+it was shown that if one were to take the type
+\ab{X}---which we use to represent an arbitrary collection of
+variable symbols---to be  the two element type, then one could combine this with our
+proof and derive a contradiction. To resolve these issues, as well as general improvements,
+we have rewritten parts of the library and have
+developed a new proof of the HSP theorem. We are confident that the new
+proof---in the \agdaalgebras library, ver.~2.0.1~\cite{ualib_v2.0.1}---is
+constructive and correct.
+
+What follows is a self-contained \agda proof of the HSP theorem.  This is done by
```

Comment:
Good edits, but I still think "achieved" is a better verb to use here instead of "done".